### PR TITLE
Appeals 995 & 996: Update `rem` value for 16px base font size

### PR DESCRIPTION
**Pull request by jamigibbs**
**Note**: Delete the description statements, complete each step. **None are optional**, but can be justified as to why they cannot be completed as written. Provide known gaps to testing that may raise the risk of merging to production.

## Summary

The global base font-size is changing from 10px to 16px to be in alignment with USWDS. This change adjusts styles with rem units for that change as these units are based off of the old base font-size.

**appeals/995**
  rootUrl: /decision-reviews/supplemental-claim/file-supplemental-claim-form-20-0995

**appeals/996**
 rootUrl: /decision-reviews/higher-level-review/request-higher-level-review-form-20-0996

main PR: https://github.com/department-of-veterans-affairs/vets-website/pull/26606

## Related issue(s)

closes https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2698

## Testing done

Changes are minimal but testing should be done with a logged-in user if necessary. I will attempt to test in the review instance, but any help from the team would be greatly appreciated.

## Screenshots

_Note: This field is mandatory for UI changes (non-component work should NOT have screenshots)._

|         | Before | After |
| ------- | ------ | ----- |
| Mobile  |        |       |
| Desktop |        |       |

## What areas of the site does it impact?

*(Describe what parts of the site are impacted **if** code touched other areas)*

### Quality Assurance & Testing

- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [ ] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [ ] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:

- [ ] The vets-website header does not contain any web-components
- [ ] I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#local-dev) to test the injected header scenario
- [ ] I reached out in the `#sitewide-public-websites` Slack channel for questions

## Requested Feedback

(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

**Review by Andrew565 — APPROVED** [review]

File: src/applications/appeals/995/sass/995-supplemental-claim.scss [hunk 1]
```diff
@@ -52,7 +52,7 @@ ul.issues,
 }
 
 article[data-location="issue-summary"] {
-  font-size: 1.5rem;
+  font-size: 0.9375rem;
   font-weight: 400;
```
Comment (Mottie) [inline src/applications/appeals/995/sass/995-supplemental-claim.scss:56]: I don't remember why this was added in the first place, but it's breaking the page layout - I think we can just remove the font-size completely

| | |
|----|----|
| With font size of `0.9375` | Screenshot 2024-04-25 at 1 34 37 PM |
| Without font size | Screenshot 2024-04-25 at 1 34 53 PM |

Comment (jamigibbs) [reply src/applications/appeals/995/sass/995-supplemental-claim.scss:56]: Thanks for finding that! I'll remove that style. 👍

**Comment by Mottie** [comment]
I'm still looking at this branch, and I'm starting to lean towards just removing all the font & margin changes - I'll either let you know specifically, or maybe just push the changes to this branch.

Also, I feel like we need to look at previous changes... the header content looks a little off in this branch

|  |  |
|----|----|
| Staging | Screenshot 2024-04-25 at 2 18 54 PM |
| Local | Screenshot 2024-04-25 at 2 19 23 PM |

- Header contact us width
- Header VA Benefits and Health Care alignment
- About VA alignment
- Form subtitle font size

**Review by Mottie — APPROVED** [review]

**Comment by jamigibbs** [comment]
@Mottie You might need to pull down the latest from this branch and run `yarn` because we had to make some updates to [the main branch](https://github.com/department-of-veterans-affairs/vets-website/pull/26606) it's based off of.

My local build looks fine (for the header) when I check it but at this point we are just asking teams to check their specific apps.

And you are welcome to make updates directly to this branch too. We will probably start working directly on [the main branch](https://github.com/department-of-veterans-affairs/vets-website/pull/26606) once this one is merged though (we realized teams would be pinged twice for PR reviews otherwise 😞 )

Screenshot 2024-04-25 at 3 18 19 PM

**Pull request merged**